hmri_getuserinput.m

```diff
@@ -1,4 +1,10 @@
-    function output = hmri_getuserinput(mainstr)
+%12 Sep 2023, written by B.E.Ugurcan
```

Comment:
We don't add dates and names. It is completely obvious from the git history who wrote what and when.

Reply:
The rule is to follow what the group does. Look at `hmri_calcA.m`, `hmri_calc_R1.m`, `hmri_calc_R2s.m`, `hmri_coreg.m`, or all the unit test files. These are all files that we have created in recent times without any name because everyone who wants to know who contributed looks at `git blame`.

We specifically don't want users to contact particular developers. Instead, we have an issue tracker for that and the group decides who is best to fix code. On the other hand, if there is an authorship attached to each file, every person who reviewed your code, and everyone who fixed issues or implemented new features will go unnoticed. Do you think this is a good solution?

Reply:
@Barisevrenugur You argued exactly my point: Credits in each file are visual noise and for files like `hmri_create_MTProt.m` they would be problematically long *today* if filled out correctly and not someday in the future. We have the README, where all developers are listed. 

@lukeje For as long as we have the hackathons, I've not seen anyone put a name in a file when we did code together. I'm sorry I mistook that for a rule we have.

Reply:
@halirutan A counter-example can be found [here](https://github.com/hMRI-group/hMRI-toolbox/blob/54a3efe289ab3725e84cd72523b7783ac1ee84da/hmri_read_vols.m), where Siawoosh did add his name.

Reply:
@lukeje Is that a counter-example? The file was created before the hackathons and just look at [how much has happened](https://github.com/hMRI-group/hMRI-toolbox/blame/master/hmri_read_vols.m) in these few lines compared to [its original state](https://github.com/hMRI-group/hMRI-toolbox/blob/ebcf3b196f43f8d980d49c6e3a61496099b6d045/hmri_read_vols.m). Putting specific names in files completely disregards the group effort that went into it.

Reply:
Of course it is a counter example: it is a file contributed at a hackathon which has a name in the header. If you want to propose a specific convention, then I recommend that you bring it up at the next hackathon. I have not seen any evidence that the norm you suggested is one that is followed in our hackathons.